docs/reference/reference-intro.mdx

```diff
@@ -24,11 +24,11 @@ slug: /reference
       <div class="card__body">
         <h3>CLI</h3>
         <p>
-          Superface CLI provides access to superface tooling from the command line.
+          Command line tool for managing Superface use-cases and providers in your projects.
```

Comment:
who cares about managing Superface use-cases? 

How about something along these lines:

"CLI for installing API integrations"